lib/coll.gi

```diff
@@ -246,15 +246,28 @@ InstallMethod( RepresentativeSmallest,
 ##  an enumerator of <C> and selects a random element of this list using the
 ##  function `RandomList', which is a pseudo random number generator.
 ##
-InstallGlobalFunction( RandomList, function(list)
-  return list[Random(GlobalMersenneTwister, 1, Length(list))];
+
+# RandomList is not a Method to avoid the (often expensive) cost of
```

Comment:
Method -> method  (only change this if you end up having to update this PR anyway; no need to trigger a full rebuild over such minor thing ;-)/

Reply:
Also, strictly speaking, I guess this should be "is not an operation", though?